Apps/Database/src/Delegates/IOutboxDelegate.cs

```diff
@@ -0,0 +1,50 @@
+// -------------------------------------------------------------------------
+//  Copyright © 2019 Province of British Columbia
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+
+namespace HealthGateway.Database.Delegates;
+
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using HealthGateway.Database.Models;
+
+/// <summary>
+/// Db delegate for outbox entity.
+/// </summary>
+public interface IOutboxDelegate
+{
+    /// <summary>
+    /// Add items to outbox
+    /// </summary>
+    /// <param name="items">Add items to the outbox queue</param>
```

Comment:
The order eventually is set by the date, and it is the concrete implementation detail, I think it's not worth changing. Let me know if you think otherwise